source/documentation/deploying_services/redis.md

```diff
@@ -193,34 +195,34 @@ Type `yes` when asked for confirmation.
 
 You can store data classified up to ‘official’ on the GOV.UK PaaS. Refer to the [data security classification documentation](/deploying_services/#data-security-classification) for more information.
 
-### Redis plans
+### Redis service plans
```

Comment:
Why is this section under 'Maintaining the service'?
I haven't found anywhere that says users have to deal with service plans regularly once they set up Redis...but maybe I've missed it?